sources/input_to_text_tree.c

```diff
@@ -31,7 +33,12 @@ int	input_to_text_tree(t_param *param)
 
 int	input_to_text_tree(t_param *param)
 {
+	char	**tokens;
+
 	(void)param;
```

Comment:
This is only for the case where the parameter is not used. Since you use it in the next line, we can remove this dummy plug.